pkgs/applications/file-managers/fm/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkgs,
+  libpanel
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "fm";
+  version = "unstable-2022-06-08";
+
+  src = fetchFromGitHub {
+    owner = "euclio";
+    repo = pname;
```

Comment:
`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21